### PR TITLE
Update host-tools documentation for 3.7, 3.8

### PR DESCRIPTION
**Pull request by sjha4**
Adding tracer and host-tools documentation from nightly to 3.7 and 3.8 documentation and 3.8 release notes.

### CODE REVIEW

File: plugins/katello/3.8/release_notes/release_notes.md [hunk 1]
```diff
@@ -23,6 +23,10 @@ It is now possible to add new environments in between an environment path
 
 Several bug fixes around Errata Management, Repositories, Docker, and more are part of this release. Please refer to the [changelog](https://github.com/Katello/katello/blob/KATELLO-3.8/CHANGELOG.md) for more details.
 
+#### Client package changes:
```
Comment (jturel) [inline plugins/katello/3.8/release_notes/release_notes.md]: This heading is too small which makes it look like it is under the Bug Fixes heading. I think you just need `##`

**Comment by jturel** [comment]
Looks good, left just one comment!

**Comment by sjha4** [comment]
Good point..Updated.. :+1:

**Comment by ehelms** [comment]
Should this also go into nightly docs for get into 3.9?

**Comment by jturel** [comment]
@ehelms this is in nightly already, but samir is backporting to relevant versions

**Comment by jturel** [comment]
ACK from me! Thanks @sjha4

**Review by jturel — APPROVED** [review]
ACK

**Pull request merged**